### PR TITLE
Fix QGIS plugin now that database entry is not in TOML

### PR DESCRIPTION
**Pull request by Huite**
This code gives an error in the QGIS plugin:

```python
    def _get_database_path_from_model_file(self) -> str:
        with open(self.path, "rb") as f:
            model_filename = tomllib.load(f)["database"]
            return str(Path(self.path).parent.joinpath(model_filename))
```

Because the database entry is no longer present in the TOML because of #815.

### CODE REVIEW

File: ribasim_qgis/widgets/dataset_widget.py [hunk 1]
```diff
@@ -260,8 +260,8 @@ def load_geopackage(self) -> None:
 
     def _get_database_path_from_model_file(self) -> str:
         with open(self.path, "rb") as f:
-            model_filename = tomllib.load(f)["database"]
-            return str(Path(self.path).parent.joinpath(model_filename))
+            input_dir = tomllib.load(f)["input_dir"]
+        return str((Path(self.path).parent / input_dir / "database.gpkg").resolve())
```
Comment (deltamarnix) [inline ribasim_qgis/widgets/dataset_widget.py:264]: Is `input_dir` always relative? Perhaps perform some more checks before you just append the paths. @visr what are the agreements on `input_dir`?

**Review by deltamarnix — CHANGES_REQUESTED** [review]

File: ribasim_qgis/widgets/dataset_widget.py (hunk 1 above)
Comment (visr) [reply ribasim_qgis/widgets/dataset_widget.py:264]: It can be absolute as well, but `/` handles that case as desired, removing the preceding part.

Comment (Huite) [reply ribasim_qgis/widgets/dataset_widget.py:266]: I tested it manually to be sure, I learned something today

Comment (Hofer-Julian) [reply ribasim_qgis/widgets/dataset_widget.py:266]: Oh nice

**Review by deltamarnix — APPROVED** [review]

**Pull request merged**